Demo/src/main/java/com/paypal/android/ui/payment/paymentbutton/PayPalButton.kt

```diff
@@ -0,0 +1,233 @@
+package com.paypal.android.ui.payment.paymentbutton
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.os.Build
+import android.util.AttributeSet
+import android.view.View
+import androidx.annotation.RequiresApi
+import androidx.core.content.res.use
+import com.paypal.android.ui.payment.paymentbutton.error.createFormattedIllegalArgumentException
+import com.paypal.pyplcheckout.R
+
+/**
+ * PayPalButton provides a PayPal button with the ability to modify the [color], [label], [shape],
+ * and [size].
+ *
+ * Setting up PayPalButton within an XML layout:
+ * ```
+ * <PayPalButton
+ *      android:id="@+id/paymentButton"
+ *      android:layout_width="match_parent"
+ *      android:layout_height="wrap_content" />
+ * ```
+ *
+ * Optionally you can provide the following attributes: `paypal_color`, `paypal_label`,
+ * `payment_button_shape`, and `payment_button_size`.
+ *
+ * Within your Activity, Fragment, or View that contains the [PayPalButton] instance you can invoke
+ * the [setup] function:
```

Comment:
this is copied from NXO right? Please remember to update this docs to match what we are going to implement on Northstar

Reply:
It is indeed, thank you for the reminder 😄